feeds/pypi/pypi_test.go

```diff
@@ -71,4 +72,7 @@ func updatesXmlHandle(w http.ResponseWriter, r *http.Request) {
 	</channel>
 </rss>
 `))
+	if err != nil {
+		fmt.Println("Unexpected error during mock http server write: %w", err)
```

Comment:
In a test this should raise the HTTP error similarly to the implementation of HttpServe outside of the tests (duplicate)